Tasks/LambdaDeployFunction/task.json

```diff
@@ -95,18 +95,15 @@
             "required": true,
             "helpMarkDown": "The runtime environment for the Lambda function you are uploading. The list of runtimes available in the pick list are those known at the time this version of the tools was released. To use a runtime not shown in the list simply enter the runtime identifier in the field.",
             "options": {
-                "dotnetcore1.0": "dotnetcore1.0",
-                "dotnetcore2.0": "dotnetcore2.0",
                 "dotnetcore2.1": "dotnetcore2.1",
                 "go1.x": "go1.x",
                 "java8": "java8",
-                "nodejs": "nodejs",
-                "nodejs4.3": "nodejs4.3",
-                "nodejs6.10": "nodejs6.10",
                 "nodejs8.10": "nodejs8.10",
-                "nodejs4.3-edge": "nodejs4.3-edge",
+                "nodejs10.x": "nodejs10.x",
                 "python2.7": "python2.7",
-                "python3.6": "python3.6"
+                "python3.6": "python3.6",
+                "python3.7": "python3.7",
+                "ruby2.5": "ruby2.5"
```

Comment:
**provided** should probably be in the list for custom runtimes.

Reply:
added